Source/WebKit/GPUProcess/mac/com.apple.WebKit.GPUProcess.sb.in

```diff
@@ -167,7 +167,7 @@
             "AppleSNBFBUserClient"))
 
     ;; QuartzCore
-    (deny IOKIT_OPEN_USER_CLIENT
+    (deny IOKIT_OPEN_USER_CLIENT (with no-report)
         (iokit-user-client-class "AGPMClient")
         (iokit-user-client-class "AppleGraphicsControlClient")
         (iokit-user-client-class "AppleGraphicsPolicyClient"))
```

Comment:
Should we also remove all three of these from `gpuIOKitClasses()` in WebPageProxy.cpp? It doesn't seem like we consume these IOKit class extensions anywhere.